### PR TITLE
suggest new research articles

### PR DESCRIPTION
**Pull request by radumarias**

### CODE REVIEW

**Comment by ericseppanen** [comment]
"The Hitchhiker’s Guide to Building an Encrypted Filesystem in Rust" was linked in last week's issue.

**Comment by radumarias** [comment]
> "The Hitchhiker’s Guide to Building an Encrypted Filesystem in Rust" was linked in last week's issue.

last week was the distributed filesystem, now it's the encrypted one

File: draft/2024-08-14-this-week-in-rust.md [hunk 1]
```diff
@@ -83,6 +87,12 @@ Some of these tasks may also have mentors available, visit the task page for mor
 <!-- * [ - ]() -->
 <!-- or if none - *No Calls for participation were submitted this week.* -->
 
+* [rencfs-desktop - Use SurrealDB](https://github.com/radumarias/rencfs-desktop/issues/3)
+* [rencfs-desktop - Implement daemon](https://github.com/radumarias/rencfs-desktop/issues/4)
+* [rencfs-desktop - Implement functionality](https://github.com/radumarias/rencfs-desktop/issues/9)
+* [rencfs-kotlin - I would like to see krencfs's custom application icon where the host operating system supports it](https://github.com/radumarias/rencfs-kotlin/issues/17)
+* [rencfs-kotlin - I would like to enforce spotless Kotlin coding standards](https://github.com/radumarias/rencfs-kotlin/issues/14)
```
Comment (cdmistman) [inline draft/2024-08-14-this-week-in-rust.md:94]:
```suggestion
```

please, rust cfps only

**Review by cdmistman — APPROVED** [review]
thanks!

**Comment by radumarias** [comment]
thanks for accepting these

**Pull request merged**